readme.md

```diff
@@ -70,3 +70,18 @@ Default: `'off'`<br>
 Values: `'on'`, `'off'`
 
 Determines whether messages from the current buffer should trigger notifications or not. This is especially useful if you use [wee-slack](https://github.com/wee-slack/wee-slack) and receive notifications for messages they send, as discussed in [#22](https://github.com/sindresorhus/weechat-notification-center/issues/22).
+
+### tags
+
+Default: `''`<br>
+Values: Comma-separated list of tags
+
+Additional message tags that can trigger notifications. This can be used in combination with `weechat.look.highlight_tags` to generate custom notifications.
```

Comment:
Can you linkify `message tags` to docs about message tag and `weechat.look.highlight_tags` to its relevant docs?